[faq] Added 'advanced osm editing' section (#229)
organicmaps/organicmaps.github.io: content/faq/editing/advanced-editing/index.md

Reviewer: advanced-map-editing

---

> title: How can I do more advanced editing in OpenStreetMap?

Author: Ok, but I would rather use "How can I do more advanced map editing?".

> description: "Tutorial for editing OpenStreetMap with more advanced tools like ID"

Reviewer: It's better to focus this page on two simple things:
1. Teach new users about OSM.
2. Recommend other editors (focus on mobile ones!). And make a link to another FAQ page "How to fix map errors using web ID editor?", it may have different recommendations, including how to copy a link to the place from OM by clicking on coordinates (there is an osm link there too), and other details.

Author: The problem is, that there are no mobile editors that are good for beginners and can do "advanced" edits (see https://github.com/organicmaps/organicmaps/pull/7748#issuecomment-2035347759). So ID on a desktop PC is the only good option.

Reviewer: What "advanced" cases for new users, in your opinion, can't be handled with existing mobile editors?

Forcing all mobile users to "find and use desktop to edit the map" doesn't look like a good idea. Most new users won't need really "advanced" stuff, from what we see, users need something simpler, that just can't be added/edited yet in OM.
At least a list of user options is needed for mobiles. I would say, that Vespucci and Go! Map can cover most of the cases.

Author: I can't say anything about Go! Map, as it's ios only, but Vespucci is rather complicated to use. But mentioning it any way is probably a good idea, as it is the only android editor that can do "advanced" stuff like adding a new road.

Osm Go! and Every Door can only edit POIs and with StreetComplete you can only answer simple quests. So with "advanced" I meant doing more than just editing POIs. So e.g. adding a new road or a building shape.

Have you tried out mobile editors so far? And if yes, how was your experience?

Reviewer: "Go! Map" is good enough for mobile. Using it to draw ways and fix areas rarely.

Then the text in the app should be clearer: "Simple map editing apps like Every Door or StreetComplete, or more advanced Vespucci" for Android and "Every Door app for simple map edits, and more advanced Go! Map"

taxonomies:
  faq: ["Editing"]

extra:
  order: 40
---

[OpenStreetMap.org](https://www.openstreetmap.org) (OSM) is a database with map data for the entire world. Similar to Wikipedia the data can be used free of charge and is crowd-sourced by millions of volunteers.

> **Organic Maps** uses the OpenStreetMap.org data to generate beautiful and detailed maps for you. 

Reviewer: Maybe add a paragraph how/when map data is updated here or below.

Author: No, I think that rather belongs into a different faq

## Found wrong or missing data in the map?

Did you spot a shop that doesn't exist? Or is your favorite restaurant not on the map? No problem, you can easily fix that! All map data used in Organic Maps comes from OpenStreetMap.org (OSM). And, as OSM is crowd-sourced, anyone, including you, can contribute and fix errors or add missing data. 

It's best to directly fix the problem yourself as described in the [Editing OpenStreetMap](#editing-openstreetmap) section. But if you don't have time for that you can just leave an [OSM Note](#osm-notes) for other contributors.

## Editing OpenStreetMap

You want to add something that's missing on the map or want to fix an error? Join the community of volunteers that make OpenStreetMap possible. For beginners, it's best to use the [ID Editor](#id-editor). If you don't have a computer with internet at hand, you can also use the basic Editor of Organic Maps that works offline.

### ID Editor

> ID is an easy-to-use, beginner-friendly editor for OpenStreetMap data. It runs in your browser, so no installation is required. You, however, need a computer, as the program is not optimized for smartphone screens. If you are new to OSM this is the editor to use.

Reviewer: Let's highlight that ID is for desktops only, and recommend other options for mobile users (they should go before details about how to use ID on the desktop.

For editing OpenStreetMap with ID follow these steps:

> 1. Create or log into your account at [OpenStreetMap.org](https://www.openstreetmap.org)

Reviewer: ```suggestion
1. Create a new account or log in at [OpenStreetMap.org](https://www.openstreetmap.org)
```

> 2.  Browse to the location you want to edit on OpenStreetMap.org and click *Edit*

Reviewer: ```suggestion
2. Browse to the location you want to edit on OpenStreetMap.org and click *Edit*
```

Author: > A direct "edit" link to osm can be inserted, right?

Using the OSMCarto map for zooming to the right location first is more convenient. On older devices zooming or moving the map in ID can be very slow on lower zoom levels.

> A hint on how to extract coordinates or osm URL from OM can be helpful

But then you have the coordinates in the clipboard of your phone and not on the desktop PC

Reviewer: We are trying to optimize here the UX for _completely new_ users, not aware of OSM, and who came from the mobile OM app. They need something really _simple_ to get involved. Later, they can learn more about OSM and about any advanced use cases.
